pkg/build/api/types.go

```diff
@@ -197,8 +197,12 @@ type DockerBuildStrategy struct {
 // STIBuildStrategy defines input parameters specific to an STI build.
 type STIBuildStrategy struct {
 	// Image is the image used to execute the build.
+	// Deprecated: will be removed in v1beta3, use From.Name.
```

Comment:
we shouldn't deprecate this, it will always be valid to just put a direct pull spec in instead of using an imagerepo ref.